src/aspire/source/image.py

```diff
@@ -88,6 +133,9 @@ def __init__(self, L, n, dtype="double", metadata=None, memory=None):
         self.generation_pipeline = Pipeline(xforms=None, memory=memory)
         self._metadata_out = None
 
+        # instantiate the accessor for the `images` property
```

Comment:
Feel weird asking you to capitalize this, but I figure someone else will if I don't :)

Reply:
That's okay i always nitpick comments and what goes around comes around